app.js

```diff
@@ -1,162 +1,241 @@
 'use strict';
+
 var userInfo = {
   //temporary hard coded values
   "spiraUrl": "https://demo.spiraservice.net/rodrigo-pereira/",
   "username": "administrator",
   "apikey": "{AA50F584-BBC9-42A0-81BA-9F8A5CD8144A}",
+  "auth": "?username=administrator&api-key={AA50F584-BBC9-42A0-81BA-9F8A5CD8144A}",
+};
+
+var currentComponents = {
+  
+};
+
+var currentUsers = {
+  //relevant users will be stored here after selecting a project
 };
 
-var newRequirement = {
+var currentReleases = {
+  //relevant releases will be stored here after selecting a project
+};
+//template objects for correct order in sheet, currently requirement
+//template only
+
+var requirementType = {
+  "Package": -1,
+  "Need": 1,
+  "Feature": 2,
+  "Use Case": 3,
+  "User Story": 4,
+  "Quality": 5,
+  "Design Element": 6,
+};
+
+var requirementObj = {
```

Comment:
keeps the keys in a specific order to match the order they would be on the excel sheet, might be reworked when export functions are reworked